content-test/common/reducers/equality.test.js

```diff
@@ -0,0 +1,35 @@
+const createStore = require("common/create-store");
+const {actions} = require("common/action-manager");
+const {createRows} = require("test/faker");
+
+describe("Reducer equality", () => {
+  let store;
+  function dispatchRows(type, length = 10) {
+    store.dispatch(actions.Response(type, createRows({length})));
+  }
+  beforeEach(() => {
+    store = createStore({logger: false});
+  });
+  it("should return a reference", () => {
+    assert.equal(store.getState(), store.getState());
```

Comment:
If I'm reading the reference at http://chaijs.com/api/assert/ correctly, .equal is non-strict equality, and .strictEqual is what we want here...